recitation_1/bash.md

```diff
@@ -177,6 +196,10 @@ echo "sum[$1..$2] = $return_val"
 
 Based on the script above, `$./sum.sh 1 5` will have an output of `15` 
 
+Acknowledgement
+------
+This was developed by TAs Nguyen Tran, Xurxo Riesco, and Mateo Maturana for COMS 3157 Spring '23

```

Comment:
Just you is fine, we are not counting reviewers as contributors

Reply:
done !